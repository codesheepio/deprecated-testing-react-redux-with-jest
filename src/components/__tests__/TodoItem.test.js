import React from 'react'
import { shallow } from 'enzyme' // eslint-disable-line import/no-extraneous-dependencies
import toJson from 'enzyme-to-json' // eslint-disable-line import/no-extraneous-dependencies
import TodoItem from '../TodoItem'

describe('TodoItem', () => {
  it('matches its snapshot - not complete', () => {
    const wrapper = shallow(<TodoItem text="Drink coffee" />)

    expect(toJson(wrapper)).toMatchSnapshot()
  })

  it('matches its snapshot - complete', () => {
    const wrapper = shallow(<TodoItem text="Drink coffee" complete />)

    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
