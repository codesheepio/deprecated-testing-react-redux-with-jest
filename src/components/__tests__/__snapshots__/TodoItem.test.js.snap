// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoItem matches its snapshot - complete 1`] = `
<li
  className="complete"
>
  Drink coffee
</li>
`;

exports[`TodoItem matches its snapshot - not complete 1`] = `
<li
  className=""
>
  Drink coffee
</li>
`;
